### DISCUSSION TITLE
The ACG website on GitHub, and accepting edit requests

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#32] [A Cosmology Group]
Since this group is public, anyone can propose changes to the ACG website.  Its structure will be maintained by reviewers so the webpage doesn't crash, but the content will _not_ be moderated.  (However, see [CONTRIBUTING.md](https://github.com/a-cosmology-group/acg/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT.md](https://github.com/a-cosmology-group/acg/blob/main/CODE_OF_CONDUCT.md).)

ACG history shows that it won't take long before a large number of cranks start taking advantage of this free platform.

Therefore I propose to:
- let anyone propose changes through pull requests,
- accept changes only after there is ***sufficient interest*** (shown by positive or negative comments),
- if, after some ***long time***, there is no interest, *prune* the branch.

The author of the changes should solicit comments on `Discussions` in a *collaborative way*.

To be defined:
- **sufficient interest** = at least ***one*** comment, or two reactions (e.g. ":+1:", ":-1:", etc.)
- **long time** = about ***one month***.

**Comment by ftessier** [comment]
I agree: 

- let people _propose_ their changes via pull requests; encourage that instead of long discussions and debates about changing content (this usually curbs a bit of the fluff 😄).

- assign reviewers from the pool of team members or repo collaborators

- assign one or two gatekeepers to merge content after review

To streamline review, in time, you can use [code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) for sections of the website that is "owned" by a designated collaborator, so that their review is automatically request for any changes that modifies the files they "own".

In your list above, I would change the first bullet to `let anyone propose changes through pull requests`, for clarity. You could set the bar at $\large n$ "👍🏻" on pull requests.

Let's see how it goes!

**Comment by RedshiftDrift** [comment]
We don't have *team members* nor *repo collaborators* 😉 
Since anybody could potentially be interested in a particular *pull request*, it's probably best to leave open the possibility of comments from anybody.
Whoever merges the content does not have to agree with it but only make sure there is enough interest.  Even if all ***n*** people say "it's garbage", that's already a sign of interest!

**Reply by ftessier** [reply]
Yes, I forgot. Everyone (with a github handle) can comment indeed. But _reviewers_ of pull requests must be members of the organization. You can see that when you click on "reviewers" on a pull request (right sidebar). In this case, the reviewers could just provide technical oversight (cleanup commits and commit messages, etc.)

**Reply by ftessier** [reply]
Note that you can still invite "external" reviewers by adding them as collaborators to the `acg` repo (under the `acg` settings), where they can have whichever role:

![image](https://github.com/a-cosmology-group/acg/assets/1444241/17d2aa6a-68f0-4a1d-a283-74646abd2f98)

**Reply by RedshiftDrift** [reply]
I haven't learned and remembered all the words yet:
- *reviewer*,
- *gatekeepers*,
- *collaborators*,
- *triage*,
- *maintain*, etc...
Let's keep this simple right now and add people as needed in the future.

**Reply by ftessier** [reply]
Sorry, by bad, indeed let's wait until these things come up in time.
